source/features/parse-backticks.tsx

```diff
@@ -13,7 +13,7 @@ function init(): void {
 		'.js-commits-list-item .mb-1, .js-commits-list-item pre', // `isCommitList` commit message and description
 		'.Details[data-issue-and-pr-hovercards-enabled] .d-none a.link-gray-dark', // `isRepoRoot` commit message
 		'.commit-title, .commit-desc', // `isCommit` commit message and description
-		'.commit-message', // `isPRConversation`, `isCompare`, `isReleasesOrTags` pushed commits
+		'.commit-message, .js-commit .pr-1 > code', // `isPRConversation`, `isCompare`, `isReleasesOrTags` pushed commits
```

Comment:
If `.commit-message` no longer appears in any of those pages, let's drop it. I don't want to keep old, unmarked selectors around forever.

Reply:
I assume older GHE versions still have this, that's why I didn't remove it.

Reply:
These selectors are always changing we are not going to keep every selector since maybe GHE is still using it.

Reply:
Can't find pushed commits in `isReleasesOrTags` pages to verify the selector. Is that no longer in GitHub or should I leave it in the comment?

Reply:
https://github.com/sindresorhus/refined-github/releases all the bullets

Reply:
That is markdown, it's highlighted already without the extension.

Reply:
Taken from https://github.com/sindresorhus/refined-github/pull/2947. I blamed the file to find the PR
https://github.com/sindresorhus/refined-github/releases/tag/20.4.1 (I think)

Reply:
https://github.com/sindresorhus/refined-github/releases/tag/20.4.1 is targeted (now at least) by `.commit-desc` and https://github.com/yakov116/TestR/releases/tag/1 is targeted by `.release-header`, so I'm going to assume that the `.commit-message` selector is definitively defunct and remove the `isReleasesOrTags` part of the comment.